app/src/main/java/org/sil/gatherwords/room/Converters.java

```diff
@@ -14,4 +18,37 @@ public static Date fromTimestamp(Long value) {
     public static Long dateToTimestamp(Date date) {
         return date == null ? null : date.getTime();
     }
+
+    /**
+     * Convert audio byte[] into a File for the media player
+     *
+     * Not necessarily a Room type converter, but still needed
+     * @param bytes The audio byte[]
+     * @return Audio file
+     */
+    public static File byteArrayToFile(byte[] bytes, String filepath) {
+        File file = new File(filepath);
+        try {
+
+            // Create the file if it doesn't exist
+            if (!file.exists()) {
+                file.createNewFile();
+            }
+
+            // Check that we can write to the file
+            if (!file.canWrite()) {
+                IOException e = new IOException("Cannot write to file");
+                Log.e("byteArrayToFile()", "Cannot write to file " + filepath, e);
+                throw e;
```

Comment:
Why do we `throw` just to catch a few lines later?  Also, you will end up with essentially doubled log message; if you wish to throw, you could just put this log message as the message in the exception.